README.md

```diff
@@ -1,42 +1,66 @@
+The following implementations are available as installable packages:
 
 | Language              | Includes         | Package name                     | Version                                                                                                                      |
 | --------------------- | ---------------- | -------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| Python                | server           | `foxglove-websocket`             | [![](https://shields.io/pypi/v/foxglove-websocket)](https://pypi.org/project/foxglove-websocket/)                            |
-| JavaScript/TypeScript | server + client  | `@foxglove/ws-protocol`          | [![](https://shields.io/npm/v/@foxglove/ws-protocol)](https://www.npmjs.com/package/@foxglove/ws-protocol)                   |
-| JavaScript/TypeScript | examples         | `@foxglove/ws-protocol-examples` | [![](https://shields.io/npm/v/@foxglove/ws-protocol-examples)](https://www.npmjs.com/package/@foxglove/ws-protocol-examples) |
+| Python                | [server](python)            | `foxglove_websocket`             | [![](https://shields.io/pypi/v/foxglove-websocket)](https://pypi.org/project/foxglove-websocket/)                            |
+| JavaScript/TypeScript | [server + client](typescript/ws-protocol)  | `@foxglove/ws-protocol`          | [![](https://shields.io/npm/v/@foxglove/ws-protocol)](https://www.npmjs.com/package/@foxglove/ws-protocol)                   |
+|   | [examples](typescript/ws-protocol-examples)         | `@foxglove/ws-protocol-examples` | [![](https://shields.io/npm/v/@foxglove/ws-protocol-examples)](https://www.npmjs.com/package/@foxglove/ws-protocol-examples) |
 | C++                   | _in development_ |
 
-## Documentation
+## Python server
 
-- [Protocol specification](docs/spec.md)
-- [Python package README](python)
-- [JavaScript/TypeScript package README](typescript/ws-protocol)
+Install the `foxglove_websocket` Python package:
 
-## Development
+```
+$ pip install foxglove_websocket
+```
 
-### Virtualenv usage
+Run a simple example server which publishes messages on a single `example_msg` topic, using JSON to encode message data and [JSON Schema](https://json-schema.org/) to describe the message layout.
 
 ```
-pipenv install --dev
-pipenv shell
+python -m foxglove_websocket.examples.simple_server
 ```
 
-### Run example servers
+To see data transmitted by your server, open [studio.foxglove.dev](https://studio.foxglove.dev) in a browser and initiate a Foxglove WebSocket connection to your WebSocket URL (`ws://localhost:8765/`).
+
+<img width="676" alt="Foxglove Studio displaying data from the example server" src="https://user-images.githubusercontent.com/14237/145260376-ddda98c5-7ed0-4239-9ce4-10778ee8240b.png">
+
+To write your own custom Python server, use the custom template provided [here](python#server-template).
```

Comment:
there are a lot of links that just say "here"... can we make the link text something more relevant/explanatory, like "Python server template"?

https://developers.google.com/style/link-text